From 72a7d00fc4ab3fe780e695b38d6ec9cb49e1cd54 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Fri, 4 Nov 2022 12:33:04 +0800
Subject: [PATCH] Revert "Linux 5.10.153"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 95aa34f72132ee42ee3f632a5540c84a5ee8624f.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d1cd7539105d..a0750d051982 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 153
+SUBLEVEL = 152
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

